### PR TITLE
Add timestamp to JSON formatter output

### PR DESCRIPTION
**Pull request by flowlo**
We're recording Selenium based tests and use this to compute time offsets to skip around in our screen recordings.

Currently there is no easy method to get an *absolute* time from Cucumber output. The only approach is to see when Cucumber starts/terminates and subtract `duration` properties from that time. This is skewed by various other things (for example browser starting up before the first step gets executed).

Summing up `duration` is quite a hassle, and can be easily circumvented by this patch. This is just a hack, and does not pass tests. Please help me finding the correct way to implement such a timestamp.

### CODE REVIEW

**Comment by charlierudolph** [comment]
I think this is solved by using scenario [attachments](https://github.com/cucumber/cucumber-js#attachments). If a scenario fails, you can attach a screenshot, or in your case a timestamp.

**Comment by charlierudolph** [comment]
Closing as the JSON formatter is actually going to get simpler and be standardized across the different cucumber implementations. Please use attachments
